### DISCUSSION TITLE
When using Pay, our my customer emails provided to Stripe or to any third-party when creating customer records?

### DISCUSSION THREAD

**Discussion by surferwat** [pay-rails/pay#796] [General] [answered]
I was a bit puzzled when I came across the note stating "Email is a field that is required by Stripe, Braintree, etc to create a Customer record." My understanding is that the customer email is not required by Stripe to create a Customer record. So, it begs the question, our my customer emails provided to Stripe or to any third-party for that matter when creating a Customer record?

URL to file: https://github.com/pay-rails/pay/blob/master/docs/1_installation.md
Text in question: "Note: Pay requires your model to have an email attribute. Email is a field that is required by Stripe, Braintree, etc to create a Customer record."

Thank you!

**Comment by excid3** [comment, marked as answer]
It used to be a required field I believe but isn't anymore. You can try it without, but there might be some code in Pay that depends on it. I know the emails do.
